lib/Transforms/SetSPIRVCapabilities.cpp

```diff
@@ -111,7 +110,11 @@ struct SetSPIRVCapabilitiesPass
 } // namespace
 
 namespace imex {
+std::unique_ptr<mlir::Pass>
+createSetSPIRVCapabilitiesPass(mlir::StringRef api) {
+  return std::make_unique<SetSPIRVCapabilitiesPass>(api);
```

Comment:
add this new ctor so that user explicitly calling this createXXX method can pass parameter.